os/dbuild.sh

```diff
@@ -465,7 +465,7 @@ function DOWNLOAD()
 {
 	# Currently supports ALL only, later this will have a menu
 	pushd ${OSDIR} > /dev/null
-	docker run --rm ${DOCKER_OPT} -v ${TOPDIR}:/root/tizenrt -w /root/tizenrt/os --privileged ${DOCKER_IMAGE}:${DOCKER_VERSION} ${BUILD_CMD} download $1 $2 $3 $4 $5 $6
+	docker run --rm -it ${DOCKER_OPT} -v ${TOPDIR}:/root/tizenrt -w /root/tizenrt/os --privileged ${DOCKER_IMAGE}:${DOCKER_VERSION} ${BUILD_CMD} download $1 $2 $3 $4 $5 $6
```

Comment:
Is there any reason to add '-it' option for download?
I think it is not necessary because there is no input after running docker container.

Reply:
common_download.sh is called in docker
In this change, I am trying to select a port by receiving user input within common_download.sh

Reply:
Yes, you're right. I missed.